src/pages/organizations/[organizationid].js

```diff
@@ -134,7 +137,7 @@ export default function Organization({ organization }) {
             icon={<ParkOutlinedIcon />}
             title="Trees Planted"
             text={organization?.featuredTrees?.total}
-            disabled={isPlanterTab ? true : false}
+            disabled={!!isPlanterTab}
```

Comment:
As in it should be like this: `disabled={isPlanterTab}`?

Reply:
> As in it should be like this: `disabled={isPlanterTab}`?
@VWRoli Yes that would be equivalent. Using `!!someVar` is for when `someVar` is not already a boolean but you need to evaluate it as one. It is similar to using `Boolean(someVar)`. In this case `isPlanterTab` will only ever be true or false so it is not needed. Using !!isPlanterTab will however ensure that the value is always cast to a boolean just in case the value was changed to something else. I still approved the pr because it's a minor difference.